pombola/south_africa/templates/interests_register/index_complete.html

```diff
@@ -3,7 +3,11 @@
 {% for person in data %}
     <h3>
         <a href="{{ person.person.get_absolute_url }}">{{ person.person.name }}</a>
-         - {{ person.year }}
+         - {% if person.source_url == "" %}
```

Comment:
A tiny thing, but In the three cases here where you've done `if foo == ""` here, it's probably clearer to do `if not foo` or reverse the `if` and `else` clauses and just do `if foo`, e.g.:

```
         - {% if person.source_url %}
             <a href="{{ person.source_url }}" target="_blank" title="View Source Document">{{ person.year }}</a>
           {% else %}
            {{ person.year }}
           {% endif %}
```